packages/gatsby-theme-store/src/pages/{StoreProduct.slug}/p.tsx

```diff
@@ -0,0 +1,45 @@
+import { graphql } from 'gatsby'
```

Comment:
not really, because this is the path. Changing the name of this file changes the route generated. So to have a product in the path `/<slug>/p` we need to have a file named `p.tsx` in our filesystem

Reply:
Yeah, i forget that gatsby uses the file's path.